css/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  background-color: #f4f5f7;
+  width: 100vw;
+  margin: 0;
+  font-family: 'poppins', sans-serif;
+}
+
+header {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  background-color: white;
+}
+
+header h4 {
+  color: #6070ff;
+  margin-left: 24px;
+  width: 100vm;
+  height: 20px;
+  font-size: 18px;
+  display: flex;
+}
+
+.nav-pic {
+  height: 12px;
+  margin-left: auto;
+  padding-right: 15px;
+}
+
+nav a {
+  display: none;
+}
+
+.home {
+  background-image: url("img/3.svg");

```

Comment:
- [ ] Please add the correct image path because the background image now it's not appearing.
[Hint]: the correct path for this image `url("../img/3.svg")`

Reply:
All image paths are fixed